### PR TITLE
Swift4

### PR DESCRIPTION
**Pull request by lightsprint09**

### CODE REVIEW

**Comment by dennispost** [comment]
Include #46

File: DBNetworkStack.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -605,6 +637,8 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.dbsystel.DBNetworkStack;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
+				SWIFT_SWIFT3_OBJC_INFERENCE = On;
```
Comment (ahersch) [inline DBNetworkStack.xcodeproj/project.pbxproj]: should be deactivated

File: DBNetworkStack.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -617,6 +651,8 @@
 				"LD_RUNPATH_SEARCH_PATHS[sdk=macosx*]" = "@executable_path/../Frameworks @loader_path/../Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = com.dbsystel.DBNetworkStackTests;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_SWIFT3_OBJC_INFERENCE = On;
```
Comment (ahersch) [inline DBNetworkStack.xcodeproj/project.pbxproj]: should be deactivated

File: DBNetworkStack.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -629,6 +665,8 @@
...
```
Comment (ahersch) [inline DBNetworkStack.xcodeproj/project.pbxproj]: should be deactivated

**Pull request merged**